app/main/forms.py

```diff
@@ -1627,7 +1627,10 @@ class AdminPreviewBrandingForm(StripWhitespaceForm):
 
 class AdminEditEmailBrandingForm(StripWhitespaceForm):
     name = GovukTextInputField("Name of brand")
-    text = GovukTextInputField("Text")
+    text = GovukTextInputField("Logo Text", param_extensions={"hint": {"text": "Text that appears beside the logo"}})
```

Comment:
Temp content or not, `Logo Text` probably doesn't need the capitalisation on the `T`. Same for alt text and the hints.

Reply:
No, I think this was intentional. CC @karlchillmaid

Reply:
This isn't temp content now, this is final. its just the service setting page stuff which karl is still playing with.

(I don't mind if karl or saima have a look just to double check capitalisation and stuff)

Reply:
oh ok. The other fields here don't have title caps? It just feels/looks inconsistent.

Reply:
No, cap case was not intentional. I forgot to raise it on Friday because we were focused on the validation errors.

```suggestion
    text = GovukTextInputField("Logo text", param_extensions={"hint": {"text": "Text that appears beside the logo"}})
```